src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -275,12 +292,16 @@ public void exportIllustrations(String query, Integer startYear, Integer endYear
         Stream<SolrDocument> docs = streamSolr(finalQuery, max);
         // Create metadata file, that has to be added to output zip
         Map<String, Object> metadataMap = makeMetadataMap(query, startYear, endYear);
+
+        Set<String> fields = new HashSet<>(Arrays.asList("pageUUID", "familyId", "lplace", "fulltext_org"));
+        StreamingOutput csvStream = SolrExport.getInstance().export(query, fields,(long) max, SolrExport.STRUCTURE.DEFAULT , SolrExport.EXPORT_FORMAT.csv );
```

Comment:
The CSV export is page oriented, but for illustration export it is ... illustration oriented. The query for CSV export should be the same as for the `exportFullpages` method.